umm-spec-lib/src/cmr/umm_spec/spatial_conversion.clj

```diff
@@ -20,30 +21,37 @@
    "WRS-1"
    "WRS-2"])
 
+(def upcase-valid-tile-identification-system-names
+  "Upcase values to allow for case-insenitive validation"
+  (map util/safe-uppercase valid-tile-identification-system-names))
+
 (defn tile-id-system-name-is-valid?
   "Return whether or not the given TileIdentificationSystemName is one of the
    valid-tile-identification-system-names"
   [tile-system-id-name]
-  (some #(= tile-system-id-name %) valid-tile-identification-system-names))
+  (some #(= (util/safe-uppercase tile-system-id-name) %) upcase-valid-tile-identification-system-names))
```

Comment:
This function is no longer needed after the generic match-enum-case.

Reply:
Fixed